articles/active-directory/saas-apps/palantir-foundry-tutorial.md

```diff
@@ -66,28 +66,17 @@ Follow these steps to enable Azure AD SSO in the Azure portal.
 
 1. In the Azure portal, on the **Palantir Foundry** application integration page, find the **Manage** section and select **single sign-on**.
 1. On the **Select a single sign-on method** page, select **SAML**.
-1. On the **Set up single sign-on with SAML** page, click the pencil icon for **Basic SAML Configuration** to edit the settings.
+1. In Foundry, download the **SAML integration metadata XML** file and save it on your computer.
 
-   ![Edit Basic SAML Configuration](common/edit-urls.png)
+   ![Download Foundry metadata file from Foundry](media/palantir-foundry-tutorial/download-metadata.png)
 
-1. On the **Basic SAML Configuration** section, perform the following steps:
+1. In the Azure portal, on the **Set up single sign-on with SAML** page, click **Upload metadata file** to upload the Palantir Foundry's XML metadata, then **Save**.
 
-    1. In the **Identifier** text box, type a value using the following pattern: `urn:uuid:<SOME_UUID>`
+   ![Upload Foundry metadata file in Azure](common/browse-upload-metadata.png)
 
-    1. In the **Reply URL** text box, type a URL using the following pattern: `https://<DOMAIN>/multipass/api/collectors/<SOME_UUID>/saml/SSO`
+1. On the **SAML Signing Certificate** section, click **Download** next to **Federation Metadata XML** to download Azure's XML metadata and save it on your computer for the **[Configure Palantir Foundry SSO](#configure-palantir-foundry-sso)** section.
 
-    1. In the **Logout URL** text box, type a URL using the following pattern: `https://<DOMAIN>/multipass/api/collectors/<SOME_UUID>/SingleLogout`
```

Comment:
Use the upload metadata file option rather than copying fields. Also, the XML file can be downloaded directly from Palantir Foundry, without contacting the Palantir support team.